### PR TITLE
Pull request for release-20.04

### PR DESCRIPTION
**Pull request by sduthil**
blog: add 20.04 sprint review
doc: upgrade notes 20.04

### CODE REVIEW

File: content/blog/sprint-review-2004.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+For more details about the aforementioned topics, please see the roadmap linked below.
+
+See you at the next sprint review!
+
+## Resources
+
+* [Install Wazo Platform](https://wazo-platform.org/install)
+
+* [Upgrade Wazo and Wazo Platform](http://documentation.wazo.community/en/stable/upgrade/upgrade.html). Be sure to read the [breaking changes](http://documentation.wazo.community/en/wazo-20.04/upgrade/upgrade_notes.html)
```
Comment (chartrandf) [inline content/blog/sprint-review-2004.md]: Links should be :
- https://wazo-platform.org/uc-doc/upgrade/
- https://wazo-platform.org/uc-doc/upgrade/upgrade_notes

File: content/uc-doc/upgrade/upgrade_notes.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
 title: Upgrade notes
 ---
 
+20.04
+=====
+
```
Comment (chartrandf) [inline content/uc-doc/upgrade/upgrade_notes.md]: Maybe we should add an anchor 
```suggestion
## <a name="20-04"></a>20.04

```

File: content/blog/sprint-review-2004.md (hunk 1 above)
Comment (fredericlepied) [reply content/blog/sprint-review-2004.md]: In fact the should be something like:

- /uc-doc/upgrade/
- /uc-doc/upgrade/upgrade_notes#20-04

Comment (fredericlepied) [inline content/blog/sprint-review-2004.md]: We need a link to the forum at the end.

Comment (pc-m) [reply content/blog/sprint-review-2004.md]: Added, thanks

**Comment by fblackburn1** [comment]
regate

**Pull request merged**